monkey/monkey_island/cc/services/config_schema.py

```diff
@@ -13,84 +13,96 @@
                     "enum": [
                         "SmbExploiter"
                     ],
-                    "title": "SMB Exploiter"
+                    "title": "SMB Exploiter",
+                    "attack_techniques": ["T1110", "T1210", "T1075"]
                 },
                 {
                     "type": "string",
                     "enum": [
                         "WmiExploiter"
                     ],
-                    "title": "WMI Exploiter"
+                    "title": "WMI Exploiter",
+                    "attack_techniques": ["T1110", "T1210"]
                 },
                 {
                     "type": "string",
                     "enum": [
                         "MSSQLExploiter"
                     ],
-                    "title": "MSSQL Exploiter"
+                    "title": "MSSQL Exploiter",
+                    "attack_techniques": ["T1110", "T1210"]
```

Comment:
It'll be best if T1110 will be linked to a single config value deciding whether to use bruteforce based exploits. we can mark for each exploit whether it's bruteforce or vulnerability, and use bruteforce ones only if the config value is set

Reply:
This will be solved on another branch